src/networking/backend.ts

```diff
@@ -55,6 +55,7 @@ export class Backend {
   async postSubscribe(
     appUserId: string,
     productId: string,
+    purchaseOptionId: string | undefined,
```

Comment:
I think this is the same and more consistent with the rest of the code

```suggestion
    purchaseOptionId?: string,
```